packages/assets-controllers/src/NftController.ts

```diff
@@ -1489,6 +1494,56 @@ export class NftController extends BaseControllerV1<NftConfig, NftState> {
     }
   }
 
+  /**
+   * Refetches NFT metadata and updates the state.
+   *
+   * @param nfts - Array of nfts
+   * @param networkClientId - The networkClientId that can be used to identify the network client to use for this request.
+   * @param userAddress - The current user address
+   */
+  async updateNftMetadata(
+    nfts: Nft[],
+    networkClientId?: NetworkClientId,
+    userAddress = this.config.selectedAddress,
+  ) {
```

Comment:
Nit: what do you think about wrapping the input arguments in an options object? This is a pattern we use throughout our controllers. 

```suggestion
   * @param options - Options for refetching NFT metadata
   * @param options.nfts - Array of nfts
   * @param options.networkClientId - The networkClientId that can be used to identify the network client to use for this request.
   * @param options.userAddress - The current user address
   */
  async updateNftMetadata({
    nfts,
    networkClientId,
    userAddress = this.config.selectedAddress,
  }: {
    nfts: Nft[];
    networkClientId?: NetworkClientId;
    userAddress?: string;
  }) {
```

Reply:
Totally, i ll fix this 🙏